### ISSUE TITLE
after deleting a game, no new games can be created 

### ISSUE THREAD

**Issue by bendixsonnenberg** [bendixsonnenberg/CS-Select#38]
1. create a game
1. terminate that game, delete that game
1. create new game, server throws error with following stacktrace:
```
java.lang.NullPointerException
	com.csselect.database.mysql.MysqlAdapter.parseTermination(MysqlAdapter.java:205)
	com.csselect.database.mysql.MysqlGameAdapter.getTermination(MysqlGameAdapter.java:161)
	com.csselect.game.Game.getTermination(Game.java:120)
	com.csselect.game.Game.isTerminated(Game.java:81)
	com.csselect.game.Game.invitePlayers(Game.java:214)
	com.csselect.game.gamecreation.GameCreator.doCreate(GameCreator.java:118)
	com.csselect.user.Organiser.createGame(Organiser.java:73)
	com.csselect.API.APIFacadeOrganiser.createGame(APIFacadeOrganiser.java:138)
	com.csselect.API.httpAPI.GameManagement.createGame(GameManagement.java:150)
	com.csselect.API.httpAPI.GameManagement.post(GameManagement.java:58)
	com.csselect.API.httpAPI.Servlet.doPost(Servlet.java:99)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:660)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
```

**Comment by bendixsonnenberg** [comment]
After trying to create the second game, the games table looks like this
![image](https://user-images.githubusercontent.com/25992157/52138715-3e894980-264e-11e9-87f2-88ddc045af3a.png)

**Comment by SeineEloquenz** [comment]
Have you reset the database before trying again?

**Comment by SeineEloquenz** [comment]
Interesting... Gonna look into it again

**Comment by bendixsonnenberg** [comment]
resolved with #40
